001-separate-secrets/proposal.md

```diff
@@ -0,0 +1,23 @@
+# Summary
+
+> Separate secrets from pipeline definitions
+
+# Motivation
+
+> At the moment secrets and pipelines are mixed together. You can insert a secret via a ((variable)).  
+> In this way you have to set and know every secret on every set-pipeline. There are ways to simplify this, like having it in a file and passing it into the set-pipeline; this makes it much more insecure, as you have to store the file somewhere and/or care for not commiting it to git.  
+
+# Proposal
```

Comment:
In terms of what this proposes so far I too am a bit confused as it sounds like today's credential management already exists in service of the same goal. There seems to be a lot of overlap between https://github.com/concourse/rfcs/pull/17#issuecomment-450441398 and things being discussed in https://github.com/concourse/rfcs/issues/5 but this RFC doesn't mention any of that.

Maybe this RFC could be steered in a more precise direction based on feedback in #5? Right now it doesn't appear to be proposing any concrete changes and seems to ignore the existing credential manager support in general.